test/JDBC/expected/BABEL-3291.out

```diff
@@ -0,0 +1,35 @@
+drop table if exists babel_3291_t1
+go
+
+create table babel_3291_t1(a1 int PRIMARY KEY, b1 int)
+go
+
+set babelfish_showplan_all on
+go
+
+select /*+SeqScan(babel_3291_t1)*/ * from babel_3291_t1 where a1 = 1
+go
+~~START~~
+text
+Query Text: select /*+SeqScan(babel_3291_t1)*/ * from babel_3291_t1 where a1 = 1
+Seq Scan on babel_3291_t1  (cost=0.00..38.25 rows=1 width=8)
```

Comment:
How do we verify that hints are actually working? For example, an empty table will *always* use a SeqScan if it's not hinted. So for these tests, at a minimum, we should also be trying to force an index scan.

Reply:
I have added the comments to clarify the expected output with and without hints